pkg/asset/machines/master.go

```diff
@@ -621,6 +621,15 @@ func (m *Master) Load(f asset.FileFetcher) (found bool, err error) {
 	}
 	m.MachineFiles = fileList
 
+	file, err = f.FetchByName(filepath.Join(directory, controlPlaneMachineSetFileName))
+	if err != nil {
+		if os.IsNotExist(err) {
+			return false, nil
```

Comment:
should this be 
```suggestion
			return true, nil
```
It seems like one of the return statements in the conditional should be opposite the other. I think this is the one that should be true but not sure ;)

Reply:
oh wait, nm, I see that the difference is in the second return argument, err vs nil